### PR TITLE
[Bromley] waste payment failed option for CSC staff

### PR DESCRIPTION
**Pull request by struan**
[skip changelog]

### CODE REVIEW

File: templates/email/default/waste/csc_payment_failed.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+  <h1 style="[% h1_style %]">There was a problem with your payment</h1>
+  <p style="[% p_style %]">There was a problem taking payment for your [% report.title %]</p>
+
+  <p style="[% p_style %]">Please contact the Customer Service Centre to try again.</p>
```
Comment (davea) [inline templates/email/default/waste/csc_payment_failed.html:16]: Might be nice to inform the user how they can do this, especially as it's sent from a no-reply email address.

File: t/app/controller/waste.t [hunk 2]
```diff
@@ -1591,6 +1591,48 @@ FixMyStreet::override_config {
...
+                current_bins => 0,
+                new_bins => 1,
+                name => 'Test McTest',
+                email => 'test@example.net'
+        } });
+        $mech->content_contains('Test McTest');
+        $mech->content_contains('£20.00');
+        $mech->content_contains('1 bin');
+        # external redirects make Test::WWW::Mechanize unhappy so clone
+        # the mech for the redirect
```
Comment (davea) [inline t/app/controller/waste.t:1611]: Can this comment (and on L1696) be removed now as the clone doesn't seem to happen?

**Review by davea — APPROVED** [review]
Couple of small things but 👍

File: templates/email/default/waste/csc_payment_failed.html (hunk 1 above)
Comment (struan) [reply templates/email/default/waste/csc_payment_failed.html:16]: I need to get Bromley to confirm all the text in the emails so this is mostly a place holder.

**Pull request merged**